### DISCUSSION TITLE
fileDropEnabled shouldn't disable native dragging inside applications

### DISCUSSION THREAD

**Discussion by Kimmottaja** [tauri-apps/tauri#5567] [Ideas]
It seems kind of weird that "fileDropEnabled: true" disables the native dragging inside tauri-applications, since both are useful features. It almost feels like it's a bug. I don't quite get why these features should be mutually exclusive. So could someone explain why it is so to me, or is this something that will be changed in the future?

**Comment by FabianLars** [comment]
This is not intentional. It's seemingly a limitation of the apis etc. It's reallyyy annoying us all so multiple team members looked at it multiple times but we were never able to fix it (if you do know of a way please reach out to us, i'd be forever grateful :D )

Other than that i _am_ working on a workaround, which works alright-ish but it has its own (pretty severe) problems so it can't be part of core, but should make a cool plugin.
